### ISSUE TITLE
Cavitating pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7090]
**Preferred term label:**
Cavitating pulmonary nodule

**Synonyms**

**Definition (free text, please give PubMed ID)**
Thick-walled abnormal gas-filled space within interstitial nodule.

**Parent term (use hpo.jax.org/app)**
pulmonary cavity

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
cavitating lung malignancy
cavitating lung coccidiomycosis
cavitating lung nocardiosis
cavitating lung cryptococcosis
granulomatosis with polyangiitis
rheumatoid nodules
pulmonary infarct

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added as HP_0034015
